supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -189,20 +223,36 @@ Always use "Administration Portal", including the capital P. When other departme
 [discrete]
 [[AMD64]]
 ==== image:images/yes.png[yes] AMD64 (noun)
-*Description*: _AMD64_ is the AMD implementation of a 64-bit version of the x86 architecture.
+*Description*: The AMD 64-bit version of the x86 architecture. Use this term for OpenShift Container Platform (OCP) attributes, Kubernetes, Operators, APIs, or CLI objects. Use this format in general sentences when referring to OCP features. 
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_This Operator is supported on AMD64 and ARM64 platforms._ 
 
 *Use it*: yes
 
-*Incorrect forms*: Hammer, x86_64, x86-64, x64, 64-bit x86
+*Incorrect forms*: _AMD64_ 
 
-*See also*:
+*See also*: xref:amd64[amd64]
+
+[discrete]
+[[amd64]]
+==== image:images/yes.png[yes] amd64 (noun)
+*Description*: The AMD 64-bit version of the x86 architecture. Use this term for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, or CLI objects. Use the lowercase format and backticks when referring to objects or parameters.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
 
-[NOTE]
-====
-The AMD64 logo is trademarked; the term _AMD64_ is not trademarked. For more information about AMD trademarks, see the http://www.amd.com/us/aboutamd/Pages/trademarks.aspx[AMD Trademark Information] page.
+Example:
 
-For more information about Intel® trademarks, see the http://www.intel.com/content/www/us/en/legal/trademarks.html[Trademark Information] and http://www.intel.com/content/www/us/en/trademarks/trademarks.html[Usage Guidelines for Customers, Licensees, and Other Third Parties] pages.
-====
+_Valid values are `amd64`._
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:AMD64[AMD64]
```

Comment:
This cross-reference link is still not working because the system sees it as identical to the [amd64] link above.

Reply:
Ah, yes, well spotted. 

See also: [AArch64](https://github.com/redhat-documentation/supplementary-style-guide/pull/295#AArch64) does not work

See also: [aarch64](https://github.com/redhat-documentation/supplementary-style-guide/pull/295#aarch64) points to Arch64 (noun)

Reply:
Yes. The problem is case-insensitivity.

Reply:
This hyperlink works for me.

Reply:
My mistake. Working for me, too. Setting to resolved.